pkgs/development/libraries/libjpeg-turbo/default.nix

```
@@ -15,11 +15,17 @@ stdenv.mkDerivation rec {
     sha256 = "0njdxfmyk8smj8bbd6fs3lxjaq3lybivwgg16gpnbiyl984dpi9b";
   };
 
-  patches =
-    stdenv.lib.optional (stdenv.hostPlatform.libc or null == "msvcrt")
+  # This is needed by freeimage
+  patches = [ ./0001-Compile-transupp.c-as-part-of-the-library.patch ]
+    ++ stdenv.lib.optional (stdenv.hostPlatform.libc or null == "msvcrt")
       ./mingw-boolean.patch;
 
-  outputs = [ "bin" "dev" "out" "man" "doc" ];
+  postFixup = ''
+    mkdir -p $dev_internal/include
+    mv $dev/include/transupp.h -t $dev_internal/include
```

```suggestion
    moveToOutput include/transupp.h $dev_private
```
And remove mkdir.

This is `outputs=[];`-friendly.